index.js

```diff
@@ -1,62 +1,16 @@
-const expect = require('chai').expect;
+const {MajorColors,MinorColors} = require("./minor_major_color");
+const GetColorFromPairNumber = require("./colorFromPairNumber.js");
 
-const MajorColors = [
```

Comment:
The archive is in a dangerous state now... the last line of the workflow still executes index.js ('node index.js'), while the test code has been moved out to another 'testing.js' file. If there is any failure in the test, the CI workflow will still succeed, since there is nothing to fail in the index.js. Switch your workflow to run testing.js and also call the toString.